tests/ha/test_pod_failure.py

```diff
@@ -3027,28 +3035,29 @@ def test_copy_object_during_pod_shutdown(self):
                                                           "Put and Get Etag mismatch")
         LOGGER.info("Step 8: Successfully download the uploaded objects & verify etags")
 
-        bucket3 = f"ha-bkt3-{int((perf_counter_ns()))}"
-        object3 = f"ha-obj3-{int((perf_counter_ns()))}"
-        bkt_obj_dict.clear()
-        bkt_obj_dict[bucket3] = object3
-        LOGGER.info("Step 9: Perform copy of %s from already created/uploaded %s to %s and verify "
-                    "copy object etags", self.object_name, self.bucket_name, bucket3)
-        resp = self.ha_obj.create_bucket_copy_obj(event, s3_test_obj=s3_test_obj,
-                                                  bucket_name=self.bucket_name,
-                                                  object_name=self.object_name,
-                                                  bkt_obj_dict=bkt_obj_dict, put_etag=put_etag,
-                                                  bkt_op=False)
-        assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 9: Performed copy of %s from already created/uploaded %s to %s and "
-                    "verified copy object etags", self.object_name, self.bucket_name, bucket3)
+        if CMN_CFG["dtm0_disabled"]:
+            bucket3 = f"ha-bkt3-{int((perf_counter_ns()))}"
```

Comment:
Can we keep this bucket already created and in here, we simply copy data from already uploaded buckets to this ?

Reply:
For copy object test, main intention is to copy object to new bucket and we have already done for 2 buckets in previous steps so doesn't make sense to do the same thing again without new bucket. This was the thought when changed this, agree @ApurwaMohite  @ankita-93

Reply:
Agree.